curated_transformers/models/embeddings.py

```diff
@@ -94,10 +94,10 @@ def forward(self, input: torch.Tensor, *, positions: Optional[Tensor] = None):
         """
         Apply rotary embeddings to an array.
 
-        input (Tensor): Array to apply the rotary embeddings to.
-        positions (Optional[Tensor]): positions of the inputs. If no
-            positions are provided, they are assumed to be [0, seq_len).
-        RETURNS (Tensor): Array with the rotary embeddings applied.
+        :param input: Array to apply the rotary embeddings to.
```

Comment:
Had to fix this up already, because I was getting errors when generating the documentation.

Reply:
This means that the docs generator will automatically pull the type and default values from the function signature?

Reply:
Not sure what is happening with the constructor documentation, let me check...

Reply:
Better:

<img width="741" alt="Screenshot 2023-05-16 at 11 13 16" src="https://github.com/explosion/curated-transformers/assets/49398/175ae39d-d0c4-4848-b2d3-952b7af576da">